src/tests/java/picard/vcf/FindMendelianViolationsTest.java

```diff
@@ -0,0 +1,148 @@
+package picard.vcf;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import htsjdk.samtools.util.IOUtil;
+import htsjdk.samtools.util.TestUtil;
+import htsjdk.tribble.index.IndexFactory;
+import htsjdk.tribble.readers.AsciiLineReader;
+import htsjdk.tribble.readers.LineIteratorImpl;
+import htsjdk.variant.vcf.VCFCodec;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picard.pedigree.Sex;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+import static picard.vcf.MendelianViolationDetector.MendelianViolation.*;
+
+/**
+ * Created by farjoun on 5/13/16.
+ */
+public class FindMendelianViolationsTest {
+
+    private static final File TEST_DATA_DIR = new File("testdata/picard/vcf");
+
+
+    @Test
+    public void testFindMedelianViolations() throws IOException {
+        final File vcfFile = new File(TEST_DATA_DIR, "NA12878.vcf");
+        final File vcfIndexFile = new File(TEST_DATA_DIR, "NA12878.vcf.idx");
+        vcfIndexFile.deleteOnExit();
```

Comment:
Why are you deleting the index file (and possibly re-creating?)?  Also, is there no way to not require the index if the program is run single-threaded?